src/components/License/UpgradeAlert.vue

```diff
@@ -7,16 +7,15 @@ export default {
     ExternalLink
   },
   computed: {
-    ...mapGetters('license', ['license']),
-    ...mapGetters('tenant', ['tenant']),
+    ...mapGetters('license', ['license', 'hasPermission']),
     isSelfServe() {
       return this.license?.terms?.is_self_serve
     },
     isUsageBased() {
       return this.license?.terms?.is_usage_based
     },
     isTenantAdmin() {
-      return this.tenant.role === 'TENANT_ADMIN'
+      return this.hasPermission('create', 'license')
```

Comment:
Not sure create is correct here.  Possibly update and/or read.

Reply:
This is currently not being used in this component. Should we delete it?